applications/ecallmgr/src/ecallmgr_fs_route.erl

```diff
@@ -347,6 +347,9 @@ start_message_handling(_Node, _FetchId, CallId, JObj) ->
 
 -spec route_req(ne_binary(), ne_binary(), wh_proplist(), atom()) -> wh_proplist().
 route_req(CallId, FetchId, Props, Node) ->
+    SwitchURL = ecallmgr_fs_node:sip_url(Node),
+    [_, SwitchURIHost] = binary:split(SwitchURL, <<"@">>),
```

Comment:
ecallmgr_fs_event_stream's version has a try/catch because of the startup sequence.
ecallmgr_fs_node may not be initialized and running (this happened when tried to initialize on startup, after connection). i had to move it to first occurence of an event. the tryt/catch here is more like a sanity check

in ecallmgr_fs_route, ecallmgr_fs_node:sip_url is already initialized so there isn't a problem here.